modules/rtm/R/gpm.R

```diff
@@ -12,6 +12,6 @@ generalized_plate_model <- function(k, refractive, N) {
     length(N) == 1
   )
   RT <- matrix(0, 2101, 2)
-  outlist <- .Fortran("gpm", k, refractive, N, RT)
+  outlist <- .Fortran("gpm", k, refractive, N, RT, PACKAGE="PEcAnRTM")
```

Comment:
@ashiklom are you still lurking in the PEcAn world? Would it be possible to get your eyes on this commit?
TL;DR: Does this patch look OK to you?

This is a proposed fix for intermittent test failures from RTM that have me puzzled for three reasons:
1. Tests are only failing on R 4.2, not on older or newer R versions.
2. Failure is traceable to .Fortran calls to `foursail` sometimes, but not always, returning NaNs. Repeated calls with the same input fail maybe 2/3 of the time but succeed often enough to make me nervous.
3. Adding the PACKAGE argument to each .Fortran call seems to fix it, which would seem to imply some kind of namespace collision... but I don't know what other package would contain a `foursail` symbol or why symbol lookup would behave nondeterministically from one invocation to another.